caluma/caluma_analytics/models.py

```diff
@@ -15,13 +15,66 @@ class AnalyticsTable(SlugModel):
 
     disable_visibilities = models.BooleanField(default=False)
     name = LocalizedField(blank=False, null=False, required=False)
-    starting_object = models.CharField(max_length=250, choices=STARTING_OBJECT_CHOICES)
+
+    starting_object = models.CharField(
+        max_length=250, choices=STARTING_OBJECT_CHOICES, null=False
```

Comment:
```suggestion
        max_length=250, choices=STARTING_OBJECT_CHOICES
```

`null=False` is the default